### PR TITLE
feat: add Instant Mix to context menu

### PR DESCRIPTION
**Pull request by noaione**
Split from the following PR: https://github.com/jellyfin/jellyfin-vue/pull/1951

### CODE REVIEW

File: frontend/src/utils/items.ts [hunk 1]
```diff
@@ -81,6 +81,16 @@ export function isValidMD5(input: string): boolean {
...
 
+/**
+ * Checks if an item is a local item.
+ *
+ * @param item - The item to be checked.
+ * @returns - A boolean representing whether the item is a local item
+ */
+export function isLocalItem(item: BaseItemDto): boolean {
+  return item?.Id?.indexOf('local') === 0;
+}
```
Comment (ferferga) [inline frontend/src/utils/items.ts:92]: Why is this needed?

File: frontend/locales/en-US.json [hunk 2]
```diff
@@ -442,6 +443,8 @@
   "signIn": "Sign in",
   "snackbar": {
     "addedToQueue": "Added to queue",
+    "instantMixFailed": "Failed to add Instant mix to queue",
```
Comment (ferferga) [inline frontend/locales/en-US.json:446]: Same thing I mentioned in #1964 about the error keys

File: frontend/locales/en-US.json [hunk 3]
```diff
@@ -559,4 +562,4 @@
   "year": "Year",
   "years": "Years",
   "youMayAlsoLike": "You may also like"
-}
+}
```
Comment (ferferga) [inline frontend/locales/en-US.json:565]: Same as #1964 about possibly Weblate getting mad at this.

**Review by ferferga — CHANGES_REQUESTED** [review]

File: frontend/src/utils/items.ts (hunk 1 above)
Comment (noaione) [reply frontend/src/utils/items.ts:92]: Legacy code? It also used in jf-web so I'm basically just adding it here

Comment (ferferga) [reply frontend/src/utils/items.ts:92]: @noaione I think we can safely remove this, if it works without it. There's no point in adding obscure code (server and web are full of them since the fork) if it works without. If later on there's a related bug with it, we can always fix it.

I recall hearing about that property before, and I think it was related for offline usage (so cases when the client had downloaded content for offline playback). That never ended up getting implemented.

We can construct some stuff thinking forward and taking into account offline support, but assuming we can't do instant mixes because the items are cached offline *it's too much*

File: frontend/locales/en-US.json (hunk 2 above)
Comment (ferferga) [reply frontend/locales/en-US.json:446]: @noaione Sorry for forgetting about telling this before: Put all the new keys in all your PRs at the root level please. I'm trying to remove a lot of duplication around keys and I found out that most of that duplication is done blindly while using subkeys.

I have a branch ready locally to remove all the nesting after all your PRs are in :)

**Review by ferferga — APPROVED** [review]
